pom.xml

```diff
@@ -329,6 +332,8 @@
                     <excludes>
                         <!-- TEMPORARY since NCE tests sometimes fail to load classes -->
                         <exclude>jmri.jmrix.nce.**</exclude>
+                        <!-- TEMPORARY since locormi tests cause later security policy issues -->
+                        <exclude>jmri.jmrix.loconet.locormi.**</exclude>
```

Comment:
Only some of the tests in this package cause any kind of issue related to the security policy, so excluding the whole package is problematic.

Some of tests that were recently added to this package ( see #3423 ) did have an issue because the security policy was not installed, but only on Maven, which was running only on the headless tests at the time, but that issue was resolved before those tests were merged.

Because we just added these tests AND we fixed this issue when the tests were installed, these tests need to be running before we can merge this in.